plugin/pkg/watch/watch.go

```diff
@@ -0,0 +1,12 @@
+package watch
+
+// Chan is used to inform the server of a change
+type Chan chan []string
+
+// Watchable is the interface watchable plugins should implement
+type Watchable interface {
+	Name() string
```

Comment:
I don't think I need this anymore, I think I had it here for a reason but ended up changing things around so I don't need it now.